src/sqlfluff/utils/reflow/reindent.py

```diff
@@ -751,14 +751,6 @@ def _map_line_buffers(
                     if "start_bracket" in elements[loc + 1].class_types:
                         continue
 
-                    # Second, check for placeholders. Indents around placeholders
-                    # are trickier to reason about. For now, don't force untaken
-                    # indents around placeholders.
-                    if "placeholder" in elements[loc + 1].class_types or (
-                        loc >= 1 and "placeholder" in elements[loc - 1].class_types
-                    ):
-                        continue
-
```

Comment:
This was only included recently (in #4499 last week), and after these changes isn't covered any more by tests. I'm removing it as no longer required.